dist/obs-server.spec

```diff
@@ -525,7 +527,6 @@ touch /srv/www/obs/api/log/production.log
 chown %{apache_user}:%{apache_group} /srv/www/obs/api/log/production.log
 
 %restart_on_update memcached
-touch /srv/www/obs/api/last_deploy
```

Comment:
@adrianschroeter why was it necessary to remove this?

Reply:
With the added 'else' this file now always get's created. So we don't have to ensure that here anymore.

@ChrisBr Do we rely on the timestamp of the last access time?

Reply:
Yes we do! We display it in the /about route.

Reply:
Yes! It is not perfect but this is also the same way e.g. capistrano / mina do it and should be good enough.
Sure when we have Salt and salt provides sth more reliable we can do it but for now I would keep it simple. And so far it was pretty reliable ....